do/loadbalancers.go

```diff
@@ -251,6 +277,30 @@ func (l *loadbalancers) buildLoadBalancerRequest(service *v1.Service, nodes []*v
 	}, nil
 }
 
+func (l *loadbalancers) waitActive(lbID string, timeout int, timespan int) (*godo.LoadBalancer, error) {
```

Comment:
If timeout and timespan are consts, no need to pass them in as func parameters. We can just call `lbActiveTimeout` from the function itself

Reply:
If the params are there for testing, feel free to add it as internal fields to `loadbalancers` struct